searx/settings.yml

```diff
@@ -746,22 +746,19 @@ engines:
 
   - name: hoogle
     engine: xpath
-    paging: true
     search_url: https://hoogle.haskell.org/?hoogle={query}&start={pageno}
     results_xpath: '//div[@class="result"]'
-    title_xpath: './/div[@class="ans"]//a'
-    url_xpath: './/div[@class="ans"]//a/@href'
-    content_xpath: './/div[@class="from"]'
-    page_size: 20
-    categories: [it, packages]
+    title_xpath: './div[@class="ans"]'
```

Comment:
search for `time`, the first link doesn't work:
* the title ends with `Uses` (because now it includes `<div class="links"><a href="https://packdeps.haskellers.com/reverse/time">Uses</a></div>` at the end)
* the link doesn't work: `https://hackage.haskell.org/package/timehttps://packdeps.haskellers.com/reverse/time`

![image](https://user-images.githubusercontent.com/1594191/150652186-b608ff87-7598-404a-9e3b-b612036aadbf.png)

Perhaps the right fix to this engine is Python code.